### DISCUSSION TITLE
Installation Problem

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#21327] [Q&A Getting Started] [answered]
MOOSE Team,

I have not used MOOSE for a long time.
I am trying to install MOOSE on my old MacBook Pro running with macOS Catalina.
Attached are the error messages.
Please help!

S. Thomas
[installation_message.docx](https://github.com/idaholab/moose/files/8922706/installation_message.docx)

**Comment by GiudGiud** [comment, marked as answer]
Hello

No need to try to run the test suite the mamba installation failed. 
Seems to be some SSL certificate error. Can you temporarily disable SSL verification, install moose packages then re-enable it?

use `conda config --set ssl_verify false`
then true

Guillaume

**Reply by milljm** [reply]
I would also love to know why Conda thinks we're using a self-signed certificate for https://conda.software.inl.gov. If you have a moment, can you try and report what the following results with?:

```bash
curl -v https://conda.software.inl.gov/public
```

**Reply by styyokuda** [reply]
Thank you, Guillaume and Milljm,

Please see the attached file and let me know what I can do.
S.Thomas
[Installaion_messages2.docx](https://github.com/idaholab/moose/files/8923823/Installaion_messages2.docx)

**Reply by styyokuda** [reply]
Guillaume and Milljm,

Despite more than 50 tests failed, the example problem of "ex01" runs correctly.
Please let me know if my installation has been done acceptably.
S. Thomas

**Reply by GiudGiud** [reply]
Hello

No 50 test failures is excessive.
Could you please attach the log with all the failures? It's usually easy to see what is going wrong from which tests failed

Guillaume
